DQM/BeamMonitor/plugins/BeamSpotDipServer.h

```diff
@@ -0,0 +1,151 @@
+#ifndef DQM_BeamMonitor__BeamSpotDipServer_h
```

Comment:
as nothing else is using this headr, so better to merge it in `BeamSpotDipServer..cc`

Reply:
We are a bit late already with this PR (and its backport).
Can this be postponed to a later PR or you think it's mandatory now?

Reply:
Thank you! I'll fix only the libraries issue that you mentioned above then!

Reply:
Added to my todo list. I will resolve this conversation.